packages/ketcher-react/src/script/ui/views/modal/components/Text/SpecialSymbols/SpecialSymbolsButton.module.less

```diff
@@ -38,21 +39,21 @@
   }
 
   &:hover {
-    background-color: @color-button-primary-clicked;
+    background-color: @color-spec-button-primary-hover;
   }
 }
 
 .textButton {
   & > svg {
-    fill: #666;
+    fill: @main-color;
     transition: 0.2s;
   }
 
   &:hover {
     background-image: none;
 
     & > svg {
-      fill: #000;
+      fill: @active-color;
```

Comment:
think, it should be #00838f here (which is @color-button-primary-hover), same as for buttons in toolbars